cmake/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Reconstruct Your project structiure:
in your `cmake` directory, you need to have `src` and `include` directories.
in `src` directory you need to have `main.cpp`, `hello_world.cpp`
in `include` directory you need to have `hello_world.h`

Reply:
+